src/bot/(de)activator.jl

```diff
@@ -0,0 +1,48 @@
+
+"""
+    precompile_activator(package_path)
+
+Activates precompile of a package by setting `should_precompile = true`
+
+`package_path` is the same as `pathof`. However, `pathof(module)` isn't used to prevent loadnig the package.
```

Comment:
```suggestion
`package_path` should be the full path to the defining file for the package, i.e., identical to `pathof(ThePkg)`. However, `pathof(module)` isn't used to prevent the need to load the package.
```